kinisi/diffusion.py

```diff
@@ -194,7 +195,7 @@ def compute_covariance_matrix(self) -> sc.Variable:
         :returns: A :py:mod:`scipp` object containing the covariance matrix.
         """
         cov = np.zeros((self.msd.data.variances.size, self.msd.data.variances.size))
-        for i in range(0, self.msd.data.variances.size):
+        for i in tqdm(range(0, self.msd.data.variances.size)):
```

Comment:
Is this cause the number of data points can be really large? I would prefer if we include it, we do it at a later date.

Reply:
Ah think this was from me messing around testing, forgot to pull from repo before starting on predict_posterior